### DISCUSSION TITLE
「开源之技」:那些「开源之道」最终呈现的部分 - 开源之道

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#33] [Announcements]
# 「开源之技」:那些「开源之道」最终呈现的部分 - 开源之道

https://opensourceway.community/posts/opensource_technology/the-art-of-open-source/

**Comment by ZoomQuiet** [comment]
#是也乎,(￣▽￣)

其实就是: 软技能 (Soft Skills: The software developer's life manual)
https://book.douban.com/subject/26835090/

- 经过学习, 可以开发, 不难
- 经过实践, 加入开源项目, 也不难
- ...

真正难的是如何公开的持续工作, 并卓越的解决一切推进过程中所有问题;
这其中, 最难的就是如何在越来越快节奏的社会中生存下去:

- 多数开源参与者, 并不是全职参与的
- 而是首先有一份普通的工作, 可以维持有尊严的普通生活
- 然后, 在业务时间, 将精力投入到自己喜爱的开源项目中, 自在探索, 自主创新, 解决自己认同的问题

此外, 还要应对普通生活中躲不开的各种问题/任务, 比如, 恋爱/婚姻/生育/教育/房产/...

可以说, 全职以开源项目为职业, 并被广泛关注讨论的, 当然是明星,
可其实, 在同时解决复数问题的普通开源参与者们, 也是明星, 自己的明星, 
自己解决一切问题, 不哭不喊, 不靠国家/公司/团队/企业/组织/....

不疯不悔的享受开源的每就位, 都是真正的明星.
